newIDE/app/src/SceneEditor/index.js

```diff
@@ -634,18 +641,25 @@ export default class SceneEditor extends React.Component<Props, State> {
 
   _onSelectInstances = (
     instances: Array<gdInitialInstance>,
-    multiSelect: boolean
+    multiSelect: boolean,
+    targetPosition?: 'center' | 'upperCenter'
   ) => {
     this.instancesSelection.selectInstances({
       instances,
       multiSelect,
       layersLocks: null,
       ignoreSeal: true,
     });
+    if (this.editorDisplay) {
+      let offset = null;
+      const { viewControls } = this.editorDisplay;
+      const viewPosition = viewControls.getViewPosition();
+      if (viewPosition && targetPosition === 'upperCenter') {
+        offset = [0, viewPosition.toSceneScale(viewPosition.getHeight() / 4)];
+      }
 
-    if (this.editor) this.editor.centerViewOnLastInstance(instances);
-    this.forceUpdateInstancesList();
-    this.forceUpdatePropertiesEditor();
+      viewControls.centerViewOnLastInstance(instances, offset);
```

Comment:
is the centering on the instance needed?
This will cause a shift in the canvas which may be confusing

Reply:
This feature has always been here so I'm keeping it.

Reply:
It's a current feature that I'm keeping, I wouldn't remove it.